doc/ledger3.texi

```diff
@@ -6430,6 +6433,9 @@ Aliases are completely ignored.
 @item --no-color
 Suppress any color TTY output.
 
+@item --no-pager
+Direct output to stdout, avoiding pager program.
+
```

Comment:
In the long run I'd like to get rid of the duplicate entries this makes maintenance cumbersome. Any idea on how to restructure the manual to avoid duplicates or using `[p][x]ref` in a meaningful way?